languagetool-core/pom.xml

```diff
@@ -171,6 +171,19 @@
             <artifactId>jackson-databind</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.measure</groupId>
```

Comment:
Please add license information for the 2 new dependencies to the 3 license.txt files we have.